src/main/scala/xebia/cakemix/AskTimeoutProvider.scala

```diff
@@ -17,7 +17,9 @@ trait AskTimeoutProvider {
 }
 
 object AskTimeoutProvider {
-  def fromConfig(config: Config, key: String): Timeout = {
+  private val AskTimeoutConfigKey = "xebia.cakemix.ask-timeout"
```

Comment:
Bit of an aesthetic argument but I would not prefix the config with "xebia" and also remove the "xebia" from the packages. I generally like short, clear packages and most users don't like overly commercial packages... but that's just my opinion. what do you think?